api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
* I think an `includehistorical` flag is a good way to add historical contracts to this call.